### PR TITLE
dominoes: Add test case with separate three-domino loops

### LINKED ISSUE #1790: Dominoes. Two three-domino loops label:"new test case idea"
Inspired by [this solution](https://exercism.io/tracks/fsharp/exercises/dominoes/solutions/ea3936878dd84c9ebc69c12a78cc1474).

```
{
      "description": "Two three-domino loops",
      "property": "canChain",
      "input": {
        "dominoes": [[1, 2], [2, 3], [3, 1], [4, 5], [5, 6], [6, 4]]
      },
      "expected": false
    },
```

### PR DESCRIPTION
**Pull request by mariohuq**
Closes #1790.

### CODE REVIEW

File: exercises/dominoes/canonical-data.json [hunk 1]
```diff
@@ -157,6 +157,22 @@
...
+    },
+    {
+      "uuid": "44704c7c-3adb-4d98-bd30-f45527cf8b49",
+      "description": "separate three-domino loops",
+      "comments": [
+        "Some solutions use nessecary but not sufficient condition:",
+        "'every stone has at least two other neighbors'.           ",
+        "It is counterexample:                                     ",
+        "one cannot chain these dominoes,                          ",
+        "but all of them satisfies stated condition.               "
```
Comment (ErikSchierboom) [inline exercises/dominoes/canonical-data.json:169]:
```suggestion
        "Some solutions only check if each stone has at least two other neighbors,       ",
        "but this test case should still fail due to the separate loops not being linked."
```

**Review by ErikSchierboom — COMMENTED** [review]
Thanks! I've suggest a slight rephrase to make the comment a bit more succinct. Feel free to adapt.

**Review by ErikSchierboom — APPROVED** [review]

**Review by petertseng — APPROVED** [review]
ordering: I was thinking place this right after b6087ff0-f555-4ea0-a71c-f9d707c5994a? But I guess it's not important.

**Comment by ErikSchierboom** [comment]
@mariohuq The location suggested by peter might indeed be a better location (might also consider using the same prefix as the disconnected test cases above that). Would you be willing to move the test case?

**Comment by mariohuq** [comment]
I will do it.

**Comment by SaschaMann** [comment]
@mariohuq when you move the test case, I think this is ready to merge :)

**Review by kytrinyx — APPROVED** [review]

**Pull request merged**